draft-ietf-add-dnr.xml

```diff
@@ -316,14 +316,14 @@
             </list></t>
         </section>
 
-        <section title="Service Mode vs ADN Only Mode">
-          <t>ServiceMode (Section 2.4.3 of <xref
-          target="I-D.ietf-dnsop-svcb-https"></xref>) SHOULD be used because
-          the Encrypted DNS options are self-contained and do not require any
-          additional DNS queries. The reader may refer to <xref
-          target="RFC7969"></xref> for an overview of advanced capabilities
-          that are supported by DHCP servers to populate configuration data
-          (e.g., issue DNS queries).</t>
+        <section title="ADN Only Mode">
+          <t>The provisioning mode, in which an ADN, a list of IP addresses,
+          and a set of service parameters of the encrypted DNS resolver are
+          supplied to a host, SHOULD be used because the Encrypted DNS options
```

Comment:
```suggestion
          supplied to a host SHOULD be used because the Encrypted DNS options
```

Reply:
Or 
```suggestion
          supplied to a host, SHOULD be used. This is because the Encrypted DNS options
```

Reply:
This what, exactly?

Reply:
@sayer, the recommended behavior is to supply the full set of provisioning information to a requesting host in DNR. That behavior may be relaxed in some contexts to allow only returning an ADN. Such mode will require extra queries.